### PR TITLE
Update domains

### PR DESCRIPTION
**Pull request by AntoineMille**

### CODE REVIEW

File: trackingfront.php [hunk 1]
```diff
@@ -81,13 +81,13 @@ public function postProcess()
...
+				$errors[] = $this->trans('login is required', array(), 'Admin.Global');
 			elseif (!Validate::isGenericName($login))
-				$errors[] = $this->l('invalid login');
+				$errors[] = $this->trans('invalid login');
 			elseif (empty($passwd))
-				$errors[] = $this->l('password is required');
+				$errors[] = $this->trans('password is required', array(), 'Admin.Global');
 			elseif (!Validate::isPasswd($passwd, 1))
-				$errors[] = $this->l('invalid password');
+				$errors[] = $this->trans('invalid password', array(), 'Admin.Global');
```
Comment (AlexEven) [inline trackingfront.php]: ('Invalid password', array(), 'Modules.Trackingfront.Shop')

This string is available only in this module so no need for a global domain.
And if used somewhere else in the back office, should be Admin.Notifications.Error, or if front content Shop.Notifications.Error

File: trackingfront.php [hunk 2]
```diff
@@ -96,7 +96,7 @@ public function postProcess()
 				FROM `'._DB_PREFIX_.'referrer`
 				WHERE `name` = \''.pSQL($login).'\' AND `passwd` = \''.pSQL($passwd).'\'');
 				if (!isset($result['id_referrer']) || !($tracking_id = (int)$result['id_referrer']))
-					$errors[] = $this->l('authentication failed');
+					$errors[] = $this->trans('authentication failed', array(), 'Admin.Global');
```
Comment (AlexEven) [inline trackingfront.php]: ('Authentication failed.', array(), 'Shop.Notifications.Error')

File: trackingfront.php [hunk 3]
```diff
@@ -183,19 +183,19 @@ public function displayAccount()
...
-			'base_fee' => $this->l('Base fee'),
-			'percent_fee' => $this->l('Percent fee'),
-			'click_fee' => $this->l('Click fee'),
-			'sales' => $this->l('Sales'),
-			'cart' => $this->l('Average cart'),
-			'reg_rate' => $this->l('Registration rate'),
-			'order_rate' => $this->l('Order rate')
+			'uniqs' => $this->trans('Unique visitors', array(), 'Modules.Trackingfront.Admin'),
+			'visitors' => $this->trans('Visitors', array(), 'Modules.Trackingfront.Admin'),
+			'visits' => $this->trans('Visits', array(), 'Modules.Trackingfront.Admin'),
```
Comment (AlexEven) [inline trackingfront.php]: Admin.Global

File: trackingfront.php [hunk 4]
```diff
@@ -183,19 +183,19 @@ public function displayAccount()
...
+			'pages' => $this->trans('Pages viewed', array(), 'Modules.Trackingfront.Admin'),
+			'registrations' => $this->trans('Registrations', array(), 'Modules.Trackingfront.Admin'),
+			'orders' => $this->trans('Orders', array(), 'Modules.Trackingfront.Admin'),
```
Comment (AlexEven) [inline trackingfront.php]: Admin.Global

File: trackingfront.php [hunk 5]
```diff
@@ -183,19 +183,19 @@ public function displayAccount()
...
+			'base_fee' => $this->trans('Base fee', array(), 'Modules.Trackingfront.Admin'),
+			'percent_fee' => $this->trans('Percent fee', array(), 'Modules.Trackingfront.Admin'),
+			'click_fee' => $this->trans('Click fee', array(), 'Modules.Trackingfront.Admin'),
+			'sales' => $this->trans('Sales', array(), 'Modules.Trackingfront.Admin'),
```
Comment (AlexEven) [inline trackingfront.php]: Admin.Global

**Comment by maximebiloe** [comment]
Thank you @AntoineMille

**Pull request merged**